### DISCUSSION TITLE
Distilled model for high-throughput Danish NLP model

### DISCUSSION THREAD

**Discussion by KennethEnevoldsen** [centre-for-humanities-computing/danish-foundation-models#200] [Missing pieces for Danish NLP]
# Statement of need
It is important to have highly performant models for Danish NLP as these are useful for processing large amount of text on limited compute budgets.

# Current status
Not in development

# Approach
- My guess is that distilling a larger model such as the dfm-encoder-large-v1 would be the best option, though it might be a better approach to simply train a small model from scratch.
- We have not looked into what the best distillation approach is.

If you wish to take on the project feel free to start a discussion here.

**Comment by KasperGroesLudvigsen** [comment]
What about quantization? Is that an undesirable approach for optimization
of processing speed in your view?

**Reply by KennethEnevoldsen** [reply]
quantisation would def. be a reasonable approach as well. Should def. be an approach to look into as well. I know there is some work on pre-quantising the model, but typically it is done during fine-tuning or after.
